servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpSetCookie.java

```diff
@@ -55,6 +55,15 @@
     @Nullable
     CharSequence expires();
 
+    /**
+     * Get the value for the
+     * <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05#section-5.3.7">SameSite attribute</a>.
+     * @return The value for the
+     * <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05#section-5.3.7">SameSite attribute</a>.
+     */
+    @Nullable
+    SameSite sameSite();
```

Comment:
note this API breaks the public APIs in a few spots

Reply:
You could also add a `default` implementation if you want to minimise breakage... not sure this is needed tho.

Reply:
Ack; I think it is OK to break APIs now.